cli/meson.build

```diff
@@ -10,6 +10,7 @@ cli_src += files(
   'interact.c',
   'log.c',
   'main.c',
+  'man_commands.c',
   'man_main.c',
```

Comment:
Can you put all code in a single `man.c` file? I don't think this warrants two separate modules.